source/developers-guide/shopware-5-upgrade-guide-for-developers/index.md

```diff
@@ -255,6 +255,21 @@ the DIC:
     * `plugin_name.filesystem.public`
     * `plugin_name.filesystem.private`
 
+### Routing
+
+SEO support for some AJAX routes defined in the template `themes/Frontend/Bare/frontend/index/index.tpl` has been removed for performance reasons. If you need SEO URLs for the following routes, you can override the block 
+`frontend_index_header_javascript` and re-enable them by removing the `_seo=false`-attribute from the `{url controller=...}`-call.
+
+The affected routes are:
+
+- `/checkout/ajaxCart`
+- `/register/index`
+- `/checkout/addArticle`
+- `/widgets/Listing/ajaxListing`
+- `/checkout/ajaxAmount`
```

Comment:
Vorsicht!
Dieser change auf /checkout/ajaxAmount verhindert das sich bei Frontend gecachten Seiten (Varnish / HTTP-Cache) die Warenkorb Menge oben rechts nicht mehr aktualisiert.

Wir haben uns jetzt damit beholfen das _seo=false in file:
Frontend/Bare/frontend/index/index.tpl: 'ajax_cart_refresh' => {url controller="checkout" action="ajaxAmount" _seo=false},
zu entfernt. Scheint aber wieder Performance zu kostet.
Müsste das ajaxAmount nicht eigentlich bei jeden CartRefresh in das localStorage des Browsers gespeichert werden?